addons/vault/aws/aws_secrets_manager_test.go

```diff
@@ -75,6 +75,28 @@ func TestAwsSecretsManagerTraitNoDefaultCreds(t *testing.T) {
 	assert.Equal(t, "false", e.ApplicationProperties["camel.vault.aws.defaultCredentialsProvider"])
 }
 
+func TestAwsSecretsManagerTraitWithSecrets(t *testing.T) {
```

Comment:
If you could add an configmap case in your test it would be nice.

Reply:
It should be done now.